dbms/src/Storages/KVStore/ProxyStateMachine.h

```diff
@@ -0,0 +1,82 @@
+// Copyright 2025 PingCAP, Inc.
```

Comment:
Just add it under KVStore/ temporarily.

Reply:
Keep this file under `KVStore` in the branch <= 7.1 is OK to me.